packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -894,6 +894,14 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
 
     private readonly dataStores: DataStores;
 
+    /**
+     * True, if GC data should be written at root of the summary tree.
+     * False, if data stores should write GC blobs in their summary tree.
+     */
+    public get writeGCDataAtRoot(): boolean {
```

Comment:
do we really need to expose this publicly on the container runtime, seems very specific and container runtime it a big part of our public api, which we should to keep as minimal as possible

Reply:
Removed it.